packages/analytics-plugin-google-global-site-tag/src/browser.ts

```diff
@@ -0,0 +1,528 @@
+          window.gtag('set', traits);
+        }
+
+        window.gtag('config', trackingId, gtagConfig);
+
+        loadedInstances[instanceName] = true;
+      }
+    },
+
+    // Set parameter scope at user level with 'set' method
+    identify: (props: IProps) => {
+      const { payload, config } = props;
+      identifyVisitor(payload.userId, payload.traits, config)
+    },
+
+    // Set parameter scope at page level with 'config' method
+    page: ({ payload, config, instance }: IProps) => {
+      if (!window.gtag || !config.trackingId) return;
+
+      const { properties } = payload;
+      const { resetCustomDimensionsOnPage, customDimensions } = config;
+      const campaign = instance.getState('context.campaign');
+
+      /* If dimensions are specified to reset, clear them before page view */
+      if (resetCustomDimensionsOnPage && resetCustomDimensionsOnPage.length && customDimensions) {
+        const resetDimensions = resetCustomDimensionsOnPage.reduce((acc: { [key: string]: unknown }, key: string) => {
+          if (customDimensions[key]) {
+            acc[key] = null // { 'dimension_name': null } etc
+          }
+          return acc
+        }, {})
+
+        if (Object.keys(resetDimensions).length) {
+          // Reset custom dimensions
+          window.gtag('set', resetDimensions);
+        }
+      }
+
+      /**
+       * Create pageview-related properties
+       */
+      const path = properties.path || document.location.pathname;
+      const pageView = {
+        page_path: path,
+        page_title: properties.title,
+        page_location: properties.url,
+      }
+
+      const campaignData = addCampaignData(campaign);
+      /**
+       * Creates property dimensions for global scope
+       */
+      const globalProperties = getProperties(properties, config.globalProperties);
+
+      const pageData = {
+        page: path,
+        title: properties.title,
+        // allow referrer override if referrer was manually set
+        referrer: properties.referrer !== document.referrer ? properties.referrer : undefined,
+        ...globalProperties,
+      }
+      window.gtag('set', pageData);
+
+      /* Set dimensions or return them for `config` if config.setCustomDimensionsToPage is false */
+      const customDimensionValues = setCustomDimensions(properties, config);
+
+      /**
+       * Creates property dimensions for user scope
+       */
+      const userProperties = getProperties(properties, config.userProperties);
+      const convertedCustomDimensions = formatCustomDimensionsIntoCustomMap(config);
+
+      /* Dimensions will only be included in the event if config.setCustomDimensionsToPage is false */
+      window.gtag('config', config.trackingId, {
+        // Every time a `pageview` is sent, we need to pass custom_map again into the configuration
+        custom_map: convertedCustomDimensions,
+        send_page_view: config.sendPageView || true,
+        ...userProperties,
+      });
```

Comment:
In my testing, all my page view events are duplicated.

Removing these lines seems to solve the issue, but I am not sure why